src/main/rx/crsf.c

```diff
@@ -248,7 +246,7 @@ bool crsfRxInit(const rxConfig_t *rxConfig, rxRuntimeConfig_t *rxRuntimeConfig)
     }
 
     rxRuntimeConfig->channelCount = CRSF_MAX_CHANNEL;
-    rxRuntimeConfig->rxRefreshRate = 11000; //!!TODO this needs checking
+    rxRuntimeConfig->rxRefreshRate = 6667; //!!TODO this needs checking
```

Comment:
Should be a constant and not a magic number. Potentially even `CRSF_TIME_BETWEEN_FRAMES_US`.

Reply:
Makes sense.  Will update and address the unit test